src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -51,91 +48,17 @@ g.test('some_binding_index_was_specified_more_than_once').fn(async t => {
 });
 
 g.test('visibility')
-  .params(
-    params()
-      .combine(poptions('type', kBindingTypes))
-      .combine(poptions('visibility', kShaderStageCombinations))
-  )
+  .cases(poptions('visibility', kShaderStageCombinations))
+  .subcases(() => poptions('entry', allBindingEntries(false)))
   .fn(async t => {
-    const { type, visibility } = t.params;
-
-    const info = kBindingTypeInfo[type];
-    const storageTextureFormat = info.resource === 'storageTex' ? 'rgba8unorm' : undefined;
+    const { visibility, entry } = t.params;
+    const info = bindingTypeInfo(entry);
 
-    const success = (visibility & ~kBindingTypeInfo[type].validStages) === 0;
+    const success = (visibility & ~info.validStages) === 0;
 
     t.expectValidationError(() => {
       t.device.createBindGroupLayout({
-        entries: [{ binding: 0, visibility, type, storageTextureFormat }],
-      });
-    }, !success);
-  });
-
-g.test('bindingTypeSpecific_optional_members')
```

Comment:
Test removed because:
- these rules are mostly no longer relevant with the new bindings style
- testing that `undefined` is _allowed_ is really an IDL test
- it makes more sense to test the _behavior_ of the defaulting in operation tests.